### DISCUSSION TITLE
New feature: automatic filter on subscriptions

### DISCUSSION THREAD

**Discussion by fadri14** [FreeTubeApp/FreeTube#4507] [General]
Hello. I'm new to the application and I'm already convinced. However, I'd like to suggest a small feature to make this application perfect. I'd like a filter to be applied automatically to the videos in the subscription list, so as to show only videos from the last 7 days, for example.

Thank you for all your hard work and I wish you all the best for the future.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @fadri14, the feature you requested for has been documented in https://github.com/FreeTubeApp/FreeTube/issues/3799.
